### ISSUE TITLE
Discovery process is scanning resources and never completed.

### ISSUE THREAD

**Issue by trtruong** [awslabs/aws-perspective#140]
I've import a region with 22K+ resources and the discovery process seems to choked and never finished.

---

**Describe the bug**
discovery process only got 1925 resources out of 22K+ and the process seems to be very long and never completed.  any guidance would be much appreciated

**To Reproduce**

1. imported region with large amount of resources
2. the discovery process seems to never end.

**Expected behavior**
expecting to see all resources in Perspective

**Screenshots**
![perspective-discovery process](https://user-images.githubusercontent.com/14808954/114101414-53477480-988b-11eb-84e9-357df044987a.png)

**Comment by svozza** [comment]
This could be due to AWS API call rate limiting as the discovery process makes a bunch of calls in parallel and obviously for so many resources that will cause issues. We've updated the discovery process in the new version (v1.1.0) to throttle the calls on our side, which should improve things. Another issue is that maybe the DB is getting overwhelmed with so many writes, could you check the CPU stats in Neptune and see what they look like?

**Comment by trtruong** [comment]
from what i can see, the DB is barely using any CPU.  at about 10%

**Comment by trtruong** [comment]
curious, when should i expect to see v1.1.0?

**Comment by svozza** [comment]
This is not guaranteed but we're aiming for around 4 weeks time.

**Comment by thebutler12** [comment]
Please create a new issue tagged with v1.1.0 if you are still experiencing this issue.
